src/components/RadioButtonGroup/RadioButtonGroup.js

```diff
@@ -14,9 +15,22 @@ const RadioButtonGroup = ({
   value: activeValue,
   name: customName
 }) => {
+  const baseStyles = ({ theme }) => css`
+    label: radio-button-group;
+    margin-bottom: ${theme.spacings.mega};
+  `;
+  const marginStyles = ({ noMargin }) =>
+    noMargin &&
```

Comment:
@felixjung @connor-baer @cristianoliveira - Can one of you help me out? This is failing linting because eslint doesn't believe I'm using the `noMargin` prop. But of course, I am.

Any tips?

Reply:
Disable the rule for this file? 😕

Reply:
Hmmmm, why haven't we had this issue before?

Reply:
I don't know honestly.